### DISCUSSION TITLE
Energiecontracten herberekenen

### DISCUSSION THREAD

**Discussion by vincedh** [dsmrreader/dsmr-reader#1475] [Questions & answers / How-to's] [answered]
Hoi Dennis,
M'n 'meterkast-pi' (die doorstuurt naar DSMR-reader op m'n nuc) en DSMR zelf is een half dagje offline geweest. Hierdoor kloppen m'n waardes in het Energiecontracten scherm niet meer (teruggeleverd is anders dan dat ik zie in het rapport van de energieleverancier wat zij hebben uitgelezen). 

Is er een manier om deze waardes opnieuw te laten berekenen, op basis van dagtotalen oid?

**Comment by dennissiemensma** [comment, marked as answer]
Bedankt voor je vraag. Ik denk dat je het beste zelf deze waardes eenmalig kan wijzigen/aanvullen bij de betreffende dagtotalen in DSMR-reader. Dit kan in de admin op de URL `/admin/dsmr_stats/daystatistics/`.

Het scherm Energiecontracten kijkt namelijk simpelweg naar alle dagtotalen van de contractperiode.

**Comment by vincedh** [comment]
Hartelijk dank weer voor je snelle antwoord, ga ik dat doen!

**Comment by dennissiemensma** [comment]
@vincedh heeft dit het uiteindelijk ook opgelost?

**Comment by vincedh** [comment]
Hoi Dennis,
Ja helemaal gelukt, tnx!

**Comment by rkollman** [comment]
Gewoon een idee. Het zou leuk zijn dat je verschillende energiecontracten invult en die dan kunt koppelen aan je voorgaande verbruik. Zo kan je dan zelf zien wat nu eigenlijk het beste voor jou situatie is om een andere energieleverancier te kiezen.

Staat zoiets toevallig al op de roadmap of vraag ik iets onhaalbaars?

**Comment by dennissiemensma** [comment]
@rkollman bedankt voor je suggestie! Ik denk dat je een leuk idee hebt, alleen is dat voor een groot deel al mogelijk via commerciele vergelijkingssites zoals gaslicht/independer/etc. 

Daarnaast zijn de (kale) tarieven slechts een deel van de kosten, maar om een goede beslissing te nemen qua leverancier zijn er ook nog andere factoren zoals contractduur, vaste kosten en eventuele kortingen. Dit zijn allemaal zaken die buiten DSMR-reader vallen, maar wel belangrijk kunnen bij je beslissing. Dus ik denk dat je uiteindelijk beter af bent bij zo'n vergelijkingssite.
